src/peep/peep.c

```diff
@@ -9021,12 +9021,12 @@ money32 set_peep_name(int flags, int state, uint16 sprite_index, uint8* text_1,
 	}
 
 	utf8* fullText = RCT2_ADDRESS(0x00F1AEF6, utf8);
-	if (flags & GAME_COMMAND_FLAG_APPLY) {
+	//if (flags & GAME_COMMAND_FLAG_APPLY) { // this check seems to be useless and causes problems in multiplayer
```

Comment:
Doesn't this stop the name from being set twice? Because the first call is to check if its possible to rename, the second call to actually rename.